localization/english/wc_laws_l_english.yml

```diff
@@ -0,0 +1,27 @@
+﻿l_english:
+ dark_authority:0 "[dark_authority|E]"
+ dark_authority_0:0 "Scattered Cults"
+ dark_authority_1:0 "Insignificant Circles"
```

Comment:
One very minor quibble, shouldn't these two be switched? Scattered cults sound more influential than insignificant circles to me. I mean, a circle is defined as a group of people, whereas a cult is defined using words like organization, religion, and system of beliefs and their adherents, which sounds more powerful than the former. 

Just my opinion though, and not very important. Thought I'd point it out anyways. Rest looks good.

Reply:
You got a point there that I'd like to support :)

Reply:
> One very minor quibble, shouldn't these two be switched? Scattered cults sound more influential than insignificant circles to me. I mean, a circle is defined as a group of people, whereas a cult is defined using words like organization, religion, and system of beliefs and their adherents, which sounds more powerful than the former.
> 
> Just my opinion though, and not very important. It works as is, just a question of terminology. Thought I'd point it out anyways. Rest looks good.

Done in 4cddafbe66b728a5dc7a360a06af9ba2cf172115.